source/adapter.js

```diff
@@ -924,12 +924,14 @@ if (navigator.mozGetUserMedia) {
     AdapterJS.WebRTCPlugin.injectPlugin();
   };
 
-  AdapterJS.WebRTCPlugin.pluginNeededButNotInstalledCb = function() {
-    AdapterJS.addEvent(document, 
-                      'readystatechange',
-                       AdapterJS.WebRTCPlugin.pluginNeededButNotInstalledCbPriv);
-    AdapterJS.WebRTCPlugin.pluginNeededButNotInstalledCbPriv();
-  };
+  if (AdapterJS.WebRTCPlugin.pluginNeededButNotInstalledCb === null) {
```

Comment:
Since this is what we do in the rest of the code, could you use the form:
a = a || function() { ... }